### ISSUE TITLE
Coinbase photo ID verification requirement means "Fund with debit/credit" is misleadingly complicated

### ISSUE THREAD

**Issue by mrstanwell** [brave/browser-laptop#4529]
When I attempted to add funds to my Brave wallet for the first time using the "Fund with debit/credit" option, I went through all the steps in the Coinbase widget to provide my name, payment info, etc.  But I got stuck on the "Transfer funds" step, and a red bar at the top of the browser window indicated that ID verification was required.  There was no link to click on to proceed further, and I had to cancel the transaction.  Since a Coinbase account had now been created for me by the widget, I tried logging into it and proceeding with ID verification.  Coinbase verification apparently requires that I upload a digital image of my government issued photo ID, using an Adobe Flash widget no less.  They require this verification before allowing any purchase or sale of BTC.

I suspect that a significant portion of Brave's target market both a) does not want to install/use Adobe Flash and b) is disinclined to upload a digital reproduction of photo ID to some database somewhere (not even Coinbase's -- they use Plaid Technology for verification, and Plaid's "privacy" statement seems to suggest they can hold data indefinitely, including after an account has been closed).  I meet both these criteria, and as such have been temporarily blocked from participating in the Brave Payments beta.

I realize you all probably have no control over Coinbase’s processes or requirements; however you might consider providing some other more palatable mechanism for funding a Brave wallet via credit card.  Or at the very least inform users up-front that the process will require uploading a photo ID via Adobe Flash, so they can decide whether or not to proceed before investing more time.

- Platform (Win7, 8, 10? macOS? Linux distro?):
  macOS 10.10.5

- Brave Version:
  0.12.3

- Steps to reproduce:
    1. Using an ID with no Coinbase account already...
    2. Try to add funds to Brave wallet using "Fund with debit/credit"

- Screenshot if needed:
I didn't get a screenshot of the original error message.  But now that I have an account with an unverified ID, whenever I try to add funds using "Fund with debit/credit" I get the following:
![screen shot 2016-10-04 at 6 12 40 pm](https://cloud.githubusercontent.com/assets/1325308/19096548/e9dc4f56-8a62-11e6-8788-4d4b52f3a0c9.png)

**Comment by diracdeltas** [comment]
i've never gotten an upload-your-ID requirement when signing up for coinbase; i'll ask what their criteria are for requiring this

**Comment by mrstanwell** [comment]
To clarify:  It's not a requirement for signing up, or even accepting/sending BTC.  It's apparently only for _buying or selling_ BTC.

**Comment by diracdeltas** [comment]
gotcha; still, i've never seen it when buying

**Comment by mrstanwell** [comment]
Hmm.  Perhaps it's a new requirement for new accounts?  I found a number of complaints in support forums, reddit, etc when they started doing this.  Their docs state: https://support.coinbase.com/customer/portal/articles/1220621

At first I thought it was just for credit card purchases.  But I linked a bank acct to my Coinbase acct for ACH transfer, but they still won't let me buy BTC without ID verification.  I have been able to successfully receive BTC from someone else though.

**Comment by johnyi275** [comment]
Hi. John from Coinbase here. Sorry about the hassle. Our intention with the Buy Widget is to provide the lowest friction UX possible (risk-adjusted) for small amounts of instant bitcoin. We've investigated this case (thanks for initiating the support ticket) and are pushing a fix that precludes the ID requirement for Buy Widget-initiated purchases. Looks like we mistakenly propagated an ID requirement meant for other UX flows to the Buy Widget, and we aim to pull this back tonight.

Purchases at Coinbase.com and our mobile apps do sometimes require ID verification (depending on the payment method, region, amount, and a host of other factors), so the requirements you saw there were likely routine (but intended for non-Buy Widget-initiated purchases, as mentioned above). Please try again in a few hours/tomorrow and let us know how it goes.

**Comment by mrstanwell** [comment]
@johnyi275, Thanks for replying.  I just reattempted, and I got the same "User is unable to buy" message in the screen-cap above.  Could my previous attempts have put a flag on my Coinbase account?  What kind of transactions should I be able to do without verification?  The "Limits" screen seems to suggest that ID uploading is a requirement for above $5k with a linked bank acct.  However, I've got a linked bank account and am being prompted for ID upload having not completed any buys/sells at all.

**Comment by johnyi275** [comment]
Apologies for the delayed response. We identified a further fix that should have lifted the ID requirement for your account. There was a somewhat complicated set of issues that triggered the requirement for you. It's hard to know whether that was before or after your last attempt so please try once more if you can. Otherwise, I think I've identified the support ticket that you sent in last week and we can confer in more detail there if that's ok.

**Comment by diracdeltas** [comment]
closing this for now since it seems to be very user-specific. thanks all.

**Comment by mrstanwell** [comment]
@johnyi275 Thanks, John.  I just tried again, and got the same "Unable to buy" message.  Not sure what complicated set of issues might have triggered that ID requirement.  I was a completely "greenfield" user.  No prior coinbase account, and I had never even bought a bitcoin before.

I'm happy to follow up on the Coinbase support system, although when I go to "Help -> My Cases" I don't see the ticket I opened.  Or the ticket I opened about not being able to see tickets I've opened.  ;-)
